packages/core/src/list-control/ListControlState.ts

```diff
@@ -51,6 +51,10 @@ export type ListControlProps<Item> = {
    * Callback used to convert an option's `value` to a string. This is needed when the value is different to the display value or the value is not a string.
    */
   valueToString?: (item: Item) => string;
+  /**
+   * If true, options will be selected on tab key press.
+   */
+  tabToSelect?: boolean;
```

Comment:
I'm wondering if this name is confusing? Maybe selectionOnTab is better

Reply:
will change this to selectionOnTab

Reply:
Can you double check with design please

Reply:
changed it to `selectOnTab` as per discussion.